week-5/B-callbacks/exercise.js

```diff
@@ -21,11 +24,14 @@ function exercise1() {
   EXERCISE 2
   =======
 
-  The function provided for you (functionCallsBack) will call the callback
-  with a result
+  A function is provided for you (functionCallsBack) that will call the first
```

Comment:
Actual function is called `functionThatCallsBack` not functionCallsBack

Reply:
Good catch. Thanks